opm/autodiff/FlowMain.hpp

```diff
@@ -386,6 +386,12 @@ namespace Opm
             std::shared_ptr<StreamLog> debugLog = std::make_shared<EclipsePRTLog>(debugFile, Log::DefaultMessageTypes);
             OpmLog::addBackend( "DEBUGLOG" ,  debugLog);
             prtLog->setMessageFormatter(std::make_shared<SimpleMessageFormatter>(false));
+            if (deck_->hasKeyword("MESSAGES")) {
```

Comment:
We really should try to limit the direct use `Deck`outside the parser code. After updates suggested in https://github.com/OPM/opm-parser/pull/921 the `Schedule` class should internalize the full time series of the MESSAGES keyword, correctly initialized with default values. No need to query the deck here in opm-simulators if it has been explicitly set.